### PR TITLE
AI-Speaker integration

### PR DESCRIPTION
**Pull request by asystentka-jolka**
## Proposed change

This PR adds documentation for the AI-Speaker integration that we are developing

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [X] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [X] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/44751
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/2121
- This PR fixes or closes issue: 

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/ai_speaker.markdown [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+then click on the `+ ADD INTEGRATION` button to add an integration,
+
+next click on **AI-Speaker**, 
+
+and finally input the host of the AI-Speaker device in your local network.
+
+<p class='img'>
+  <img src='/images/integrations/ai_speaker/add_ais_integraion.png' alt='Screenshot: Add AIS Integration'>
+</p>
```
Comment (frenck) [inline source/_integrations/ai_speaker.markdown]: Please use Markdown instead

File: source/_integrations/ai_speaker.markdown [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+
+<div class='note'>
+You can use this integration in basic functionality without the AI-Speaker gateway / loudspeaker. All you need to do is install the free Android <a href="https://play.google.com/store/apps/details?id=pl.sviete.dom&hl=pl&gl=US">AIS-dom application from Google Play</a> and turn on the player function in it.
```
Comment (frenck) [inline source/_integrations/ai_speaker.markdown]: Please add a new line before opening and closing tag of the div and replace the links with Markdown.

File: source/_integrations/ai_speaker.markdown [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+## Configuration
+
+In Home Assistant go to **Menu** -> **Configuration** -> **Integrations**
+
+then click on the `+ ADD INTEGRATION` button to add an integration,
+
+next click on **AI-Speaker**, 
+
+and finally input the host of the AI-Speaker device in your local network.
```
Comment (frenck) [inline source/_integrations/ai_speaker.markdown]:
```suggestion
In Home Assistant go to **Menu** -> **Configuration** -> **Integrations**
then click on the `+ ADD INTEGRATION` button to add an integration,
next click on **AI-Speaker**
and finally, input the host of the AI-Speaker device in your local network.
```

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/ai_speaker.markdown (hunk 1 above)
Comment (asystentka-jolka) [reply source/_integrations/ai_speaker.markdown]: ok, thanks, done

File: source/_integrations/ai_speaker.markdown (hunk 2 above)
Comment (asystentka-jolka) [reply source/_integrations/ai_speaker.markdown]: good to know - thanks :) , done

File: source/_integrations/ai_speaker.markdown (hunk 3 above)
Comment (asystentka-jolka) [reply source/_integrations/ai_speaker.markdown]: done in commit 34f24f4

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.
⚠️ The `ha_release` might need an update before merging this one in.

**Comment by frenck** [comment]
Closing PR as the parent PR has been closed.

**Pull request closed without merging**